lib/hammer_cli_katello/content_view.rb

```diff
@@ -285,7 +285,8 @@ class AddContentViewVersionCommand < HammerCLIKatello::AddAssociatedCommand
 
       option "--content-view-id", "CONTENT_VIEW_ID",
         _("Content view numeric identifier to search by"),
-        :attribute_name => :option_content_view_version_content_view_id
+        :attribute_name => :option_content_view_id,
```

Comment:
Mixed hash syntax :man_shrugging: not enforced in our projects

Reply:
I was looking at other hash syntax uses in the project. I was confused about your comment until I realized that the project uses both. The example I found used the hash rocket. Should we us the new syntax in hammer? I would like to be a good citizen and do the right thing. :)